app/javascript/packs/commentDropdowns.js

```diff
@@ -1,4 +1,5 @@
 import { addSnackbarItem } from '../Snackbar';
+import { initializeMemberMenu } from '../topNavigation/utilities';
```

Comment:
This file is related to the top member navigaton, and isn't something we need to use here - trying to repurpose this is part of what is causing the stacking issues

Instead, see below, and comments in `app/views/articles/_full_comment_area.html.erb` for how we can initialize the dropdown